### DISCUSSION TITLE
各级标题导致正文排版被纵向拉伸，各段落之间间距过大

### DISCUSSION THREAD

**Discussion by dreistein543** [nju-lug/NJUThesis#176] [Q&A] [answered]
似乎模板为了美观考虑，对各级标题设置了类似word中“与下段同页”以及不会单独出现在页面最后一行，
然而这种设置在遇到
（1）插入表格、图片导致该页内容相对较少时，
（2）连续小标题在一起，且恰好临近内容较少的上一页时
可能会强行要求小标题在新一页，而对原本页面的内容纵向拉伸，使页面各段落之间纵向间距过大。

不知是否有办法控制这种纵向拉伸？（即，依旧保留小标题自动下一页，但上一页宁可底端留白，也不希望段落间距过大）

**Comment by note286** [comment, marked as answer]
`\raggedbottom`

**Reply by dreistein543** [reply]
可以略微详细说明一下该命令如何使用吗？是放在导言区、还是放在需要控制的段落后面？

**Reply by note286** [reply]
导言区末端即可。

**Reply by dreistein543** [reply]
谢啦，我去尝试一下
